src/main/java/io/reactivesw/category/domain/service/CategoryService.java

```diff
@@ -134,7 +136,8 @@ public CategoryView updateCategory(String id, Integer version, List<UpdateAction
     //TODO send message, if slug be updated
     CategoryView result = CategoryMapper.toModel(updatedEntity);
 
-    LOG.debug("End updateCategory, updated Category is {}.", result);
+    LOG.debug("Exit. Updated category, updated category id is {}.", result.getId());
```

Comment:
remove "Updated category".
"CategoryId: {}." is ok.
use "id" to instead of "result.getId()".